linux/step05_centos6_py27_ius_nginx.sh

```diff
@@ -30,11 +30,16 @@ set -u
 # Install OMERO.web requirements
 file=~omero/OMERO.server/share/web/requirements-py27-nginx.txt
 
-# introduced in 5.2.0
+file=~omero/OMERO.server/share/web/requirements-py27-all.txt
+
+# introduced in 5.3.0
 if [ -f $file ]; then
 	#start-latest
 	/home/omero/omeroenv/bin/pip2.7 install -r $file
 	#end-latest
+elif $(is_less_than $OMEROVER 5.3); then
```

Comment:
the various switches in place are mainly there is someone need to "build" a 5.1, etc server
happy to remove them since previously we did not have branches for that repo only develop